docs/spark/tutorials/get-started.md

```diff
@@ -197,13 +195,28 @@ This .NET app counts words with Apache Spark
 
 2. Run the following command to submit your application to run on Apache Spark:
 
-   ```powershell
-   %SPARK_HOME%\bin\spark-submit --class org.apache.spark.deploy.dotnet.DotnetRunner --master local bin\Debug\netcoreapp3.0\microsoft-spark-2.4.x-0.6.0.jar dotnet bin\Debug\netcoreapp3.0\mySparkApp.dll
+   ```console
+   spark-submit \
+   --class org.apache.spark.deploy.dotnet.DotnetRunner \
+   --master local \
+   microsoft-spark-2.4.x-<version>.jar \
+   dotnet HelloSpark.dll
    ```
 
-3. When your app runs, the word count data of the *input.txt* file is written to the console.
+   > [!NOTE]
+    > This command assumes you have downloaded Apache Spark and added it to your PATH environment variable to be able to use `spark-submit`. Otherwise, you'd have to use the full path (for example, *c:\bin\apache-spark\bin\spark-submit* or *~/spark/bin/spark-submit*).
+
+3. The output of the application should look similar to the output below:
 
-Congratulations! You successfully authored and ran a .NET for Apache Spark app.
+   ```console
+   +----+-------+
+   | age|   name|
+   +----+-------+
+   |null|Michael|
+   |  30|   Andy|
+   |  19| Justin|
+   +----+-------+
+   ```
 
 ## Next steps
```

Comment:
Add next steps? Maybe link to batch processing (now that you can run a .NET for Spark app on any platform, take a look at how to write your first app)

Reply:
Actually, when previewing the file, I see that it does have next steps (they just didn't appear here), so they should be fine as-is (not sure why I can't see them here?)

Reply:
@bamurtaugh, yeah, that happens to me as well. I cannot see the next steps linking in Edit view but File view.

Reply:
Next steps are already there, they just weren't changed from the original article. You're only seeing updates to the original.

Reply:
Yep that makes sense, thanks for the explanation!

Reply:
Makes sense. Sounds good. Thanks!